sysutils/smart/src/www/widgets/widgets/smart_status.widget.php

```diff
@@ -1,43 +1,43 @@
 <?php
 
 /*
-	Copyright (C) 2014 Deciso B.V.
-	Copyright 2012 mkirbst @ pfSense Forum
-	All rights reserved.
+ *   Copyright (C) 2014 Deciso B.V.
```

Comment:
thanks, almost... it's " * Copy..." so 1 space not 3

Reply:
done